test/utils/runners.go

```diff
@@ -584,7 +585,16 @@ func (config *RCConfig) create() error {
 
 	config.applyTo(rc.Spec.Template)
 
-	_, err := config.Client.CoreV1().ReplicationControllers(config.Namespace).Create(rc)
+	var err error
```

Comment:
Instead of that, can you please:
1. promote retryWithExponentialBackoff function to this file: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/scalability/load.go#L607 (and make it public)
- or even to framework?
2. use that function instead here
3. add retries also in other place in this file (for all Create() requests for pods, RCs)
4. add retries in load.go where they are missing (we are at least missing them for services creation and deletion).

Reply:
Good point about making it a utility function that can be used elsewhere - I'll try doing it.
Btw - wdyt about a generic function (across object kinds) that does this kind of creation with retries? Should I try to maybe add one such function in client library?

Reply:
If you can somehow generalize it - great.

Regarding putting it into client-library, I'm slightly against it. Although in many cases it might make sense, whether you want to retry or not depends on your usecase. And we don't want to implicitly suggest people that they should always use it.

Reply:
Ok.. For now I'll take the (individual functions for kinds + changing it in our test framework) approach.

Reply:
Done - PTAL